dom-calculator-starter-code/js/calc.js

```diff
@@ -0,0 +1,71 @@
+//make variables from class/id
+
+var buttonNum = document.getElementsByClassName("buttonNum");
+var buttonOperator = document.getElementsByClassName("operator");
+var display = document.getElementsByClassName("screen");
+var equalsButton = document.getElementsByClassName("equals");
+var buttonReset = document.getElementsByClassName("buttonClear");
+var firstnumber, secondnumber, operator;
```

Comment:
You can't define variables like this, they should be on separate lines e.g.:

var firstnumber;
var secondnumber;
var operator;